app/config/assetic.yml

```diff
@@ -63,8 +63,6 @@ assetic:
         - '%kernel.root_dir%/../web/assets/app/css/_history.css'
         - '%kernel.root_dir%/../web/vendor/sweetalert/dist/sweetalert.css'
         - '%kernel.root_dir%/../web/vendor/fancybox/source/jquery.fancybox.css'
-        - 'https://formbuilder.online/assets/css/form-builder.min.css'
```

Comment:
@tugrulcan you thought awesome ;) but, we using this style assets on project. can you replace with existing ones